### PR TITLE
Change the application to include views and not be api-only

### PR DESCRIPTION
**Pull request by EGiataganas**
### Description of change

Change the application to include moves and not be API-only

### CODE REVIEW

File: app/javascript/controllers/hello_controller.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { Controller } from "@hotwired/stimulus"
```
Comment (notapatch) [inline app/javascript/controllers/hello_controller.js:1]: You might consider removing it

File: app/javascript/controllers/index.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+// This file is auto-generated by ./bin/rails stimulus:manifest:update
+// Run that command whenever you add a new controller or create them with
+// ./bin/rails generate stimulus controllerName
+
+import { application } from "./application"
+
+import HelloController from "./hello_controller"
+application.register("hello", HelloController)
```
Comment (notapatch) [inline app/javascript/controllers/index.js:8]: You might consider removing it (though it is a nice template)

**Review by notapatch — APPROVED** [review]
LGYM

File: app/javascript/controllers/hello_controller.js (hunk 1 above)
Comment (EGiataganas) [reply app/javascript/controllers/hello_controller.js:1]: We might need it later 😄

**Pull request merged**